lib/geokit-rails/acts_as_mappable.rb

```diff
@@ -149,6 +149,48 @@ def farthest(options = {})
         by_distance(options).last(1)
       end
 
+      def geo_scope(options = {})
+        
+        # Scope method replaced with .where(nil) method
+        # Check issue list https://github.com/rails/rails/issues/12756
+        
+        arel = self.is_a?(ActiveRecord::Relation) ? self : self.where(nil) # self.scoped
+        #p "--opt--#{options}"
+        origin  = extract_origin_from_options(options)
+        units   = extract_units_from_options(options)
+        formula = extract_formula_from_options(options)
+        bounds  = extract_bounds_from_options(options)
+        
+        if origin || bounds
+          bounds = formulate_bounds_from_distance(options, origin, units) unless bounds
+          if origin
+            @distance_formula = distance_sql(origin, units, formula)
+
+            if arel.select_values.blank?
+              star_select = Arel::SqlLiteral.new(arel.quoted_table_name + '.*')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.